travis_pages.js

```diff
@@ -44,9 +44,17 @@ module.exports = function(sender){
 
 			exec('sudo gem install travis', (error, stdout, stderr) => {
 				if( error != null){
-					console.error( colors.rainbow(sPrefix + " => ") + "oh BBoy! There is some issue on installing. Try installing travis gem first and then retry. ")
-					process.exit();
+					console.error( colors.rainbow(sPrefix + " => ") + "oh BBoy! There is some issue on installing. Try for windows version ")
+					exec('gem install travis', (error, stdout, stderr) => {
```

Comment:
This will make installation very time-consuming. If by any case installation is not working due to any other error.
What we can do is detect the OS environment from node then will make installation command using the environment type ( windows or Linux ) and then execute it. 
Making it simpler to work and understand.